src/components/Reports/ReportView.tsx

```diff
@@ -65,6 +65,28 @@ export default function ReportView(): JSX.Element {
     }
   };
 
+  const [initialReportFiles, setInitialReportFiles] = useState<ReportFile[]>();
+  useEffect(() => {
+    const getFiles = async () => {
+      if (report) {
+        const fileListResponse = await ReportService.getReportFiles(report.id);
+        if (!fileListResponse.requestSucceeded || fileListResponse.error) {
```

Comment:
same.. also looks like an opportunity for a hook (to use across view and edit.. with that one difference of updating the updatedFiles state).. just a thought.. not necessary

Reply:
That's a good idea. Saves some code repetition